Bottom: ccfd54d88cfdf5e5446872f405aec178a13d8e4b
Top:    0e34c6d4d9bc53674f1dc141ca7da6d4136fbee3
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 15:34:01 +0800

vhost_net: add interrupt coalescing support

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 7d137a4..9740534 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -415,6 +415,7 @@ static void handle_tx(struct vhost_net *net)
 		}
 	}
 out:
+	vhost_check_coalesce_and_signal(vq->dev, vq);
 	mutex_unlock(&vq->mutex);
 }
 
@@ -638,6 +639,7 @@ static void handle_rx(struct vhost_net *net)
 		}
 	}
 out:
+	vhost_check_coalesce_and_signal(vq->dev, vq);
 	mutex_unlock(&vq->mutex);
 }
